### DISCUSSION TITLE
Best way to mute chime?

### DISCUSSION THREAD

**Discussion by kcaref** [dgreif/ring#1361] [Q&A]
My hub (a HomePod mini) is located in my son's room, and as such I would like to mute the chime that occurs upon someone ringing my buzzer. Can anyone recommend a simple way to mute or disable it? Thanks in advance!

**Comment by smockle** [comment]
In Apple’s Home app, find the tile for the HomePod mini and long-press it to open the accessory menu. Tap the “Accessory Settings” menu item, then scroll down to “Doorbell Chime”. Tap “Doorbell Chime”, locate the toggle next to your HomePod mini, then toggle it off.

**Reply by kcaref** [reply]
Thanks for the reply! Unfortunately I don't have a "Doorbell Chime" option in my HomePod mini settings. In retrospect it would make perfect sense for there to be one, but for some reason it appears to be hidden.

**Reply by kcaref** [reply]
After a bit of poking around the Home app I've realized that my Ring device is being classified as a Lock, not a Doorbell. Maybe that's why the chime option doesn't appear? And if that's the case, is there a way to reclassify it?

**Reply by tsightler** [reply]
It's not completely clear to me how this is possible.  Locks don't cause chimes so clearly something is causing the HomePod to chime and that can pretty much only be a doorbell.  What Ring device do you have?  Is it an intercom perhaps?

**Reply by kcaref** [reply]
Yes! It's the Ring Intercom. Sorry I didn't clarify that earlier...

**Reply by tsightler** [reply]
Both doobell cameras and intercoms register an observable characteristic with the doorbell service, that's how notifications arrive and the clearly is working or you wouldn't get any chime at all.  I would expect there to be an accessory on the device for the doorbell state.

**Reply by kcaref** [reply]
The plugin creates two accessories: One is a Lock, which allows me to trigger the buzzer to open the lobby door of my building, and the other is a Switch with an unconfigured button. I'm not sure why the latter appears, but I suppose it is to allow for additional actions to be taken upon triggering the buzzer (e.g., turning off music or whatever). But regardless, neither of these accessories has a visible setting for doorbell state.

**Reply by tsightler** [reply]
The code as written registers both a doorbell and a programmable switch as a service.  I assume, although I do not know this as I'm not that familiar with the Homekit side of things, that the doorbell is what causes things like the chimes, while the promgrammable switch can be used to trigger automations, but I'm not sure.  I don't know why the option does not appear in Homekit, but perhaps it never does or perhaps it only appears if the device is a camera, which an intercom is not.

**Comment by nico81** [comment]
Any chance to get chime muted, or at least have the intercom classified as a doorbell on Apple Home?

**Reply by tsightler** [reply]
Isn't this already possible by simply setting the volume of the chime to 0?  Regarding doorbell, the intercom device is already registered with Homekit as a doorbell, but it seems Apple doesn't provide the chime disable function unless there is also a camera, I have no idea why.

**Comment by kcaref** [comment]
Thanks for following up! The issue is that the device is classified as a
lock in Homekit, not a doorbell. Thus there is no option to mute or
otherwise adjust the volume of the chime.

**Reply by tsightler** [reply]
Sorry, I thought you were referring to the Ring chime, which definitely has volume control.  But now I think I see you were just referring to homepods chime.

Unfortunately, I don't know how to solve this issue.  Clearly the code registers a doorbell service, and this represented by the fact that it chimes at all.  Maybe it's because the lock is being set as the primary service?  Unfortunately, as none of the maintainers have an Intercom, it's quite difficult to troubleshoot, but we'd be happy to entertain a PR that addresses the issue.
